src/cart/crt_utils.c

```diff
@@ -372,31 +490,44 @@ crtu_cli_start_basic(char *local_group_name, char *srv_group_name,
 	rc = pthread_create(progress_thread, NULL, crtu_progress_fn, crt_ctx);
 	D_ASSERTF(rc == 0, "pthread_create() failed; rc=%d\n", rc);
 
-	if (use_cfg) {
-		while (attach_retries-- > 0) {
-			rc = crt_group_attach(srv_group_name, grp);
-			if (rc == 0)
+	if (! use_daos_agent_env) {
```

Comment:
(style)  space prohibited after that '!' (ctx:BxW)

Reply:
a7a392ebeb634c4bd428fa52186e852442752056